src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -56,7 +58,7 @@ public static ConversationStore getTestInstance(PersistentStorageAgent persisten
   private PersistentStorageAgent persistentStorageAgent;
 
   /** The in-memory list of Conversations. */
-  private List<Conversation> conversations;
+  private static List<Conversation> conversations;
```

Comment:
I would advise against doing this. Static variables mean we can only run a single instance per process, which will break tests.